### PR TITLE
Add DRAFT for Shay

### PR DESCRIPTION
**Pull request by wpschaub**
EDO Ce would like to plan how to track and empower engineering to increase deployment frequency, and reduce lead time for change, MTTR, and change failure rates.

### CODE REVIEW

**Comment by wpschaub** [comment]
@ShayVannery here is your DRAFT. Please update the PR or send me revisions :)

**Review by ShayVannery — DISMISSED** [review]

**Review by erwsch1 — DISMISSED** [review]

**Comment by wsbctechnicalblog** [comment]
2022.04.29 Publication merge

File: content/improving-dora-metrics.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+1. Skill/Knowledge constraints
+2. Process constraints
+3. Budgetary constraints
+4. Architecture/ Infrastructure limitations
+
+---
+
+# 6.0 Conclusion
+
+It is likely that some of the barriers we encountered can be eliminated by coaching team members on existing organizational protocols and widely accepted engineering best practices around development, testing, deployment, monitoring, error detection and mitigation. Communities of Practice must make contin
```
Comment (ShayVannery) [inline content/improving-dora-metrics.md]: Communities of Practice must make continuous learning a key priority, and time must be allocated for learning and teaching.

Comment (AndreKaminski) [reply content/improving-dora-metrics.md]: Looks good to me

**Pull request merged**